src/wazuh_modules/vulnerability_scanner/src/vulnerabilityScannerFacade.hpp

```diff
@@ -58,6 +61,14 @@ class VulnerabilityScannerFacade final : public Singleton<VulnerabilityScannerFa
      */
     void stop();
 
+    /**
+     * @brief Decompress database content.
+     *
+     * @param databaseVersion Current database version.
+     * @return true if the decompression was executed.
```

Comment:
Nitpick: I think a more accurate description would be 
```suggestion
     * @return true if the decompression was sucesfull.
```